GETTING_STARTED.md

```diff
@@ -0,0 +1,368 @@
+}
+```
+
+Each value for such a multi-language text column is a JSON object like this:
+
+```
+{
+  "de": "Deutsch",
+  "en": "English",
+  "es": "Español",
+  "fr": "Français"
+}
+```
+
+The keys of such a multi-language value object are called language tags ([RFC 5646 Tags for Identifying Languages](https://tools.ietf.org/html/rfc5646)). Here are some examples:
+
+* `de` represents German.
+* `de-DE` represents German (`de`) as used in Germany (`DE`).
+* `de-AT` represents German (`de`) as used in Austria (`AT`).
+* `en` represents English.
+* `en-US` represents English (`en`) as used in United States (`US`).
+
+In the majority of cases multi-language data is provided for a specific language and not for a country specific language. This in most GRUD instances there is no `de-DE` or `de-AT` but a `de` for both because the language differences are not that big.
+
+Additionally to multi-language there is the `languageType` `country`. If a column has this language type there is also another field called `countryCodes`. Here is an example of a multi-country column definition:
+
+```
+{
+  "id": 2,
+  "name": "price",
+  "kind": "currency",
+  "languageType": "country",
+  "countryCodes": [
+    "DE",
+    "AT",
+    "US"
+  ]
+  // ...
+}
+```
+
+Each value for such a multi-country currency column is a JSON object like this:
+
+```
+{
+  "DE": 47.11,
+  "AT": 47.11,
+  "US": 40.99
+}
+```
+
+#### `link`
+
+This is probably the most important column kind. It is used to make a relation between two tables. A link is a uni- or bidirectional association. Here is an example of a link column definition:
+
+```
+{
+  "id": 8,
+  "name": "country",
+  "identifier": false,
+  "kind": "link",
+  "toTable": 1, // foreign table ID
+  "toColumn": { // foreign column definition
+    "id": 1,
+    "name": "name",
+    "identifier": true,
+    "kind": "shorttext",
+    "languageType": "language",
+    "multilanguage": true
+    // ...
+  }
+  // ...
+}
+```
+
+A link column always points to a specific table — in this case the table `1`. A link value is the association between at least two rows. In the following example there is one association, one link to the foreign row with the ID `13`. The value used here is the `identifier` value of the foreign row as example in the data retrieval chapter. To retrieve the full row you can call `GET /tables/1/rows/13`.
+
+```
+[
+  {
+    "id": 13,
+    "value": {
+      "cs": "Česká republika",
+      "de": "Tschechien",
+      "en": "Czech Republic",
+      "en-US": "Czech Republic",
+      "es": "Chequia",
+      "fr": "République Tchèque",
+      "it": "Repubblica Ceca",
+      "pl": "Czechy"
+    }
+  }
+]
+```
+
+#### `concat` and `group`
+
+This column kinds combines multiple columns into one column. The column kind `concat` is only used to combine multiple `identifier` columns into one column. Additionally there is the column kind `group` which can be used to combine multiple columns into one column for convenience.
```

Comment:
Maybe an example would clarify this a bit better. Setting multiple `identifier` columns in table A will create a `concat` column which is used as value in the links. The `group` column lets you combine multiple columns int one, for example grouping three columns "height", "length" and "depth" together into a single field for the UI as "<height> x <length> x <depth>".

Reply:
Are concat columns created automatically when a table contains more than one identifier column, or do you have to create them manually?